en/CommandLine.md

```diff
@@ -117,6 +118,17 @@ java -jar JabRef.jar -o path\split,xmp  path\entries.bib -n
 ```
 
 
+### Import BibTeX: `-importBibtex`
+
+Import or load code directly from the BibTeX file.
+
+This only works for BibTeX files, and does not support files of other import formats.
+
+If the GUI is not suppressed (using the `-n` option), any imported or loaded BibTeX file will show up in the main window.
+
+*Note:* The `-importBibtex` option can be specified only once, and for one file only.
+
+
```

Comment:
Could you remove this empty line?

(and add an empty line below the next heading? --> Markdown-Lint Rule 22: https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022

Reply:
@koppor I have readjusted the empty lines such that there are in accordance with markdown lint rules. I noticed that the other headings in this file don't quite follow the rule though. Shall I readjust the empty lines around the other headings as well, or just leave them alone? Please let me know. Thank you.